src/App.js

```diff
@@ -3,27 +3,33 @@ import './App.css';
 import Colors from './colors'
 
 class ColorApp extends Component {
-  state = {
-    color: [
-      {color: this.getRandomColor(),
-       lock: true
-      },
-      {color: this.getRandomColor(),
-        lock: true
-      },
-      {color: this.getRandomColor(),
-        lock: true
-      },
-      {color: this.getRandomColor(),
-        lock: true
-      },
-      {color: this.getRandomColor(),
-        lock: true
-      },
-    ]
+
+  constructor(){
+    super();
+    let state = {
+      colors: [
+        {color: this.getRandomColor(),
+         lock: false
+        },
+        {color: this.getRandomColor(),
+          lock: false
+        },
+        {color: this.getRandomColor(),
+          lock: false
+        },
+        {color: this.getRandomColor(),
+          lock: false
+        },
+        {color: this.getRandomColor(),
+          lock: false
+        },
+      ]
+    }
+    this.state = state
   }
+  
 
```

Comment:
I added a constructor and super even though you are not using props in this component it still good practice to get use the using them. NOTE: all components in production environments(aka job) do not constructors. More than likely if there is a constructor within a component that component is a class component.